tests/utils.go

```diff
@@ -42,6 +55,16 @@ func CDIFailHandler(message string, callerSkip ...int) {
 	Fail(message, callerSkip...)
 }
 
+func GetCDIClient() *clientset.Clientset {
+
+	cfg, err := clientcmd.BuildConfigFromFlags(master, kubeconfig)
+	PanicOnError(err)
+	cdiClient, err := clientset.NewForConfig(cfg)
+	PanicOnError(err)
+
+	return cdiClient
```

Comment:
Similar to k8s, I'd like to see an e2e framework struct that contains ns, client(s), provider, maybe some storage labels, etc so that these values are available to all tests. See https://github.com/kubernetes/kubernetes/blob/master/test/e2e/framework/framework.go#L136 and `BeforeEach` just below.  Unless, there's a good kubevirt reason not to...